.eslintrc.js

```diff
@@ -176,5 +176,8 @@ module.exports = {
 		// - events because we use it for some event emitters
 		// - path because we use it quite a bit
 		'import/no-nodejs-modules': [ 'error', { allow: [ 'url', 'events', 'path', 'config' ] } ],
+
+		// Ensure we do not reintroduce non-inclusive language into our repository
+		'inclusive-language/use-inclusive-words': 'error',
```

Comment:
A better place for this rule definition would be the `eslint-config-wpcalypso` package. It's reusable in other projects and the `/.eslintrc.js` file contains only overrides that are specific to the Calypso app in particular.